l10n_ch_mis_reports/data/mis_report_kpi_pl.xml

```diff
@@ -0,0 +1,245 @@
+<odoo noupdate="1">
+
+    <record id="mis_report_kpi_0001" model="mis.report.kpi">
+        <field name="name">CH_30</field>
+        <field name="description">Chiffre d'affaires</field>
+        <field
+            name="expression"
+        >-balp[('code', '>=', '3'),('code', '&lt;', '39')][]</field>
+        <field name="style_id" ref="style_details" />
```

Comment:
Thanks for your work.
Could you juste change the style here with
```suggestion
        <field name="style_id" ref="style_header" />
```
Otherwise it's all ok for me.

Reply:
Thanks for your review. Indeed it's nicer with the header style.